contrib/babelfishpg_tsql/src/pltsql_bulkcopy.c

```diff
@@ -588,6 +629,13 @@ ExecuteBulkCopy(BulkCopyState cstate, int rowCount, int colCount,
 							RelationGetRelationName(cstate->rel))));
 	}
 
+	/* Track all incoming Datums to be able to free them after the flush */
+	for (int i = 0; i < rowCount * colCount; i++)
+	{
+		cstate->bufferedValues = lappend(cstate->bufferedValues, (void *) Values[i]);
+		cstate->bufferedValueAllocFlags = lappend_int(cstate->bufferedValueAllocFlags, ValueAllocFlags[i] ? 1 : 0);
```

Comment:
Why cant we directly store booleans in cstate->bufferedValueAllocFlags instead of integers?

Reply:
Just wanted to be explicit with `T_IntList` usage. Updated this line to remove int conversion.